.expeditor/scripts/shared.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Download public and private keys for the "core" origin from Builder.
+#
+# Currently relies on a global variable `hab_binary` being set, since
+# in the Linux build process, we need to switch binaries mid-way
+# through the pipeline. As we bring more platforms into play, this may
+# change. FYI.
+import_keys() {
+    echo "--- :key: Downloading 'core' public keys from ${HAB_BLDR_URL:-Builder}"
+    ${hab_binary:?} origin key download core
+    echo "--- :closed_lock_with_key: Downloading latest 'core' secret key from ${HAB_BLDR_URL:-Builder}"
+    ${hab_binary:?} origin key download \
+        --auth="${HAB_AUTH_TOKEN}" \
+        --secret \
+        core
+}
+
+# Always install the latest hab binary appropriate for your linux platform
+#
+# Assumes you have `BUILD_PKG_TARGET` set
+install_latest_stable_hab_binary() {
+    echo "--- Installing latest hab binary for $BUILD_PKG_TARGET using curl|bash"
+    # TODO:
+    # really weird corner case on linux2 because the 0.82.0 versions of both
+    # are the same. let's just delete it
+    rm -rf /hab/pkgs/core/hab/0.82.0
+    curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash -s -- -t "$BUILD_PKG_TARGET"
+    hab_binary="/bin/hab"
+    declare -g hab_binary
+    echo "--- :habicat: Installed latest stable hab: $(${hab_binary} --version)"
+}
+
+# The following get/set functions abstract the meta-data key
+# names to provide consistant access, taking into account the
+# target, where appropriate.
+
+get_hab_ident() {
+    local target=$1
```

Comment:
The `local` usage is 😚👌 

We should still quote these variable expansions, though. Observe the behavior of this script:
```bash
#!/bin/bash

echo without quoting: $1
echo with quoting: "$1"
```
If you call it like
```bash
$ bash the-script.sh "*"
```